localized/en-us/games/eggquiz/eggquizmath_levels.tsv

```diff
@@ -389,37 +388,37 @@ SpecialTest_21	1	210001, 210002, 210003, 210003, 210004
+			240002	word_problem	·UUUUÜ, Îé·UUUæÙ ×ð¢ ¿æßÜ ·ðUUU 7 Õñ» ÍðÐ ¥æÁ, Îé·UUUæÙ ×ð¢ ¿æßÜ ·ðUUU 2 Õñ» ãñ¢Ð Îé·UUUæÙ ×ð¢ ç·UUUUÌÙð Õñ» ¿æßÜ çÕ·UUUU »°?	कल दुकान में चावल के ७ बैग थे। आज दुकान में चावल के २ बैग हैं। दुकान में कितने बैग चावल बिक गए?				5				
+			240003	word_problem	ÇðçßÇ ¥æñÚ ©â·ðUUU ÎôSÌ 9 ÂÌ¢» ©Ç¸æÌð ãñ¢Ð ·éUUUÀ ÂÌ¢» °·UUUU ÂðÇ¸ ×ð¢ È¢UUUâ ÁæÌè ãñ¢Ð 6 ÂÌ¢»ð¢ ¥Öè Öè ©Ç¸ Úãè ãñ¢Ð ÂðÇ¸ ×ð¢ ç·UUUUÌÙè ÂÌ¢»ð¢ ¥Å·UUUè ãñ¢?	डेविड और उसके दोस्त ९ पतंग उड़ाते हैं। कुछ पतंग एक पेड़ में फस जाती हैं। ६ पतंगें अभी भी उड़ रही हैं। पेड़ में कितनी पतंगें अटकी हैं?				3				
+			240004	word_problem	°·UUUU ¥æÎ×è »éyÕæÚð Üð Áæ Úãæ Íæ Üðç·UUUUÙ ãßæ Ùð 6 ©Ç¸æ çÎ°Ð ©â·ðUUU Âæâ 2 »éyÕæÚð Õ¿ð ãñ¢Ð ©â·ðUUU Âæâ ÂãÜð  ç·UUUUÌÙð »éyÕæÚô¢ Íð?	एक आदमी गुब्बारे ले जा रहा था लेकिन हवा ने ६ उड़ा दिए। उसके पास २ गुब्बारे बचे हैं। उसके पास पहले  कितने  गुब्बारे थे?				8				
 			240005	word_problem	·éUUUÀ ¥æ× Åô·UUUUÚè ×ð¢ ÍðÐ Åô·UUUUÚè âð 4 ¥æ× çÙ·UUUæÜð »° ÍðÐ ¥Õ 6 ¥æ× ãñ¢Ð Åô·UUUUÚè ×ð¢ ÂãÜð ç·UUUUÌÙð ¥æ× Íð?	कुछ आम टोकरी में थे। टोकरी से ४ आम निकाले गए थे। अब ६ आम हैं। टोकरी में पहले कितने आम थे?				10				
 			240006	word_problem	°·UUUU ÂðÇ¸ ·UUUè àææ¹æ ÂÚ 8 Âÿæè ÕñÆðÐ ·éUUUÀ ©Ç¸ »°Ð ¥Õ àææ¹æ ÂÚ 3 Âÿæè ãñ¢Ð ç·UUUUÌÙð Âÿæè ©Ç¸ »°?	एक पेड़ की शाखा पर ८ पक्षी बैठे। कुछ उड़ गए। अब शाखा पर ३ पक्षी हैं। कितने पक्षी उड़ गए?				5				
 			240007	word_problem	ç×Øæ Ùð ƒæÚ ·ðUUU ÕÙð Áæ× ·ðUUU 10 ÁæÚ ÕæÁæÚ ×ð¢ Üæ°Ð ©âÙð ·éUUUÀ ÁæÚ Õð¿ð ¥æñÚ 3 ÁæÚ Õ¿ð ÍðÐ ©âÙð ÕæÁæÚ ×ð¢ ç·UUUUÌÙð ÁæÚ Õð¿ð?	मिया ने घर के बने जाम के १० जार बाजार में लाए। उसने कुछ जार बेचे और ३ जार बचे थे। उसने बाजार में कितने जार बेचे?				7				
-			240008	word_problem	çÅ× ·ðUUU ·éUUU}æð Ùð ·éUUUÀ çÂËÜô¢ ·UUUUô Á‹× çÎØæÐ çÅ× Ùð ¥ÂÙð ÎôSÌô¢ ·UUUUô 6 çÂËÜð çÎ°Ð ¥Õ ©â·ðUUU Âæâ 2 çÂËÜð Õ¿ð ãñ¢Ð ©â·ðUUU ·éUUU}æð Ùð ç·UUUUÌÙð çÂËÜô¢ ·UUUUô Á‹× çÎØæ?	टिम के कु8ो ने कुछ पिल्लों को जन्म दिया। टिम ने अपने दोस्तों को ६ पिल्ले दिए। अब उसके पास २ पिल्ले बचे हैं। उसके कु8ो ने कितने पिल्लों को जन्म दिया?				8				
-			240009	word_problem	ÁðÙ Ùð ¥ÂÙð ÂçÚßæÚ ·ðUUU çÜ° 6 âñ¢Ççß¿ ÕÙæ°Ð ©â·ðUUU ÂçÚßæÚ Ùð ·éUUUÀ ¹æØæ ¥æñÚ 2 Õ¿ð ÍðÐ ©â·ðUUU ÂçÚßæÚ Ùð ç·UUUUÌÙð âñ¢Ççß¿ ¹æ°?	जेन ने अपने परिवार के लिए ६ सैंडविच बनाए। उसके परिवार ने कुछ खाया और २ बचे थे। उसके परिवार ने कितने सैंडविच खाए?				4				
+			240008	word_problem	çÅ× ·ðUUU ·éUUU}æð Ùð ·éUUUÀ çÂËÜô¢ ·UUUUô Á‹× çÎØæÐ çÅ× Ùð ¥ÂÙð ÎôSÌô¢ ·UUUUô 6 çÂËÜð çÎ°Ð ¥Õ ©â·ðUUU Âæâ 2 çÂËÜð Õ¿ð ãñ¢Ð ©â·ðUUU ·éUUU}æð Ùð ç·UUUUÌÙð çÂËÜô¢ ·UUUUô Á‹× çÎØæ?	टिम के कुते ने कुछ पिल्लों को जन्म दिया। टिम ने अपने दोस्तों को ६ पिल्ले दिए। अब उसके पास २ पिल्ले बचे हैं। उसके कुते ने कितने पिल्लों को जन्म दिया?				8				
+			240009	word_problem	ÁðÙ Ùð ¥ÂÙð ÂçÚUßæÚU ·Ô¤ çÜ° 6 âñ´Ççß¿ ÕÙæ°Ð ©â·Ô¤ ÂçÚUßæÚU Ùð ·¤éÀ ¹æ° ¥õÚU 2 Õ¿ð ÍðÐ ©â·Ô¤ ÂçÚUßæÚU Ùð ç·¤ÌÙð âñ´Ççß¿ ¹æ°?	जेन ने अपने परिवार के लिए ६ सैंडविच बनाए। उसके परिवार ने कुछ खाए और २ बचे थे। उसके परिवार ने कितने सैंडविच खाए?				4				
```

Comment:
`और` in the Chanakya encoding is not correct

Reply:
Resolved